### ISSUE TITLE
Pickle old knownnodes missing/incomplete error handler

### ISSUE THREAD

**Issue by PeterSurda** [Bitmessage/PyBitmessage#1662]
This happened during upgrade:

```
Traceback (most recent call last):
  File "/usr/src/PyBitmessage/src/bitmessagemain.py", line 485, in <module>
    main()
  File "/usr/src/PyBitmessage/src/bitmessagemain.py", line 481, in main
    mainprogram.start()
  File "/usr/src/PyBitmessage/src/bitmessagemain.py", line 238, in start
    readKnownNodes()
  File "/usr/src/PyBitmessage/src/network/knownnodes.py", line 158, in readKnownNodes
    pickle_deserialize_old_knownnodes(source)
  File "/usr/src/PyBitmessage/src/network/knownnodes.py", line 84, in pickle_deserialize_old_knownnodes
    knownNodes = pickle.load(source)
  File "/usr/lib/python2.7/pickle.py", line 1378, in load
    return Unpickler(file).load()
  File "/usr/lib/python2.7/pickle.py", line 858, in load
    dispatch[key](self)
  File "/usr/lib/python2.7/pickle.py", line 1090, in load_global
    klass = self.find_class(module, name)
  File "/usr/lib/python2.7/pickle.py", line 1126, in find_class
    klass = getattr(mod, name)
AttributeError: 'module' object has no attribute 'Peer'
```

maybe the `pickle.load` at line 84 of knownnodes.py should be wrapped in try/except.

**Comment by g1itch** [comment]
Oops. d56191e was probably caused new bug

**Comment by g1itch** [comment]
I also suspect that I broke the process of upgrade from versions before 0.6.2 in other places. The upgrade process should probably have it's own test suite.

**Comment by PeterSurda** [comment]
fixed in 9fe4ad0489ca42801e06abf05333a42213c3c9c8
